docs/sources/articles/networking.md

```diff
@@ -452,6 +455,21 @@ address, you can edit your system-wide Docker server settings and add the
 option `--ip=IP_ADDRESS`.  Remember to restart your Docker server after
 editing this setting.
 
+> **Note**:
+> With hairpin NAT enabled (`--userland-proxy=false`), containers port exposure
+> is achieved purely through iptables rules, and no attempt to bind the exposed
+> port is ever made. This means that nothing prevents shadowing a previously
+> listening service outside of Docker through exposing the same port for a
+> container. In such conflicting situation, Docker created iptables rules will
+> take precedence and route to the container.
```

Comment:
If I understand correctly, this note will be removed after https://github.com/docker/docker/issues/13024 is implemented (which will in the same 1.7 release as this PR)

(If it is, I don't think we should invest too much time in perfectioning the text)